docs/architecture/managers/mining_manager.md

```diff
@@ -0,0 +1,117 @@
+<<<<<<< HEAD
```

Comment:
It seems you had some conflicts in here. :stuck_out_tongue:

Reply:
Shame on me :see_no_evil: